examples/3d/fog.rs

```diff
@@ -0,0 +1,310 @@
+//! This example shows how to use distance fog
```

Comment:
Describing the controls here would be nice. Full keybindings in a single little table would be great.